include/egg/core/eggColorFader.h

```diff
@@ -3,16 +3,32 @@
 #include <egg/core/eggFader.h>
 
 #include <nw4r/ut/Color.h>
+#include <nw4r/ut/Rect.h>
 
 namespace EGG {
 
 class ColorFader : public Fader {
 public:
     ColorFader(f32, f32, f32, f32, nw4r::ut::Color, Fader::EStatus);
-    virtual ~ColorFader();
+    virtual ~ColorFader() {}
+    virtual void setStatus(EStatus status);
+    virtual EStatus getStatus() const {
+        return mStatus;
+    }
+    virtual bool fadeIn();
+    virtual bool fadeOut();
+    virtual bool calc();
+    virtual void draw();
+
+    void setFrame(u16 frame);
+    void setColor(nw4r::ut::Color color);
 
 private:
-    u8 _08[0x24 - 0x08];
+    u8 mFlags;
```

Comment:
`mFlags` is actually an instance of `TBitFlag<u8>`:

```
9] __ct__Q23EGG10ColorFaderFffffQ34nw4r2ut5ColorQ33EGG5Fader7EStatus (func,global) found in eggND.a D:\home\Project\RevoYawaraka\egg\egg\modules\core\src\eggColorFader.o
    . . . . .
    10] __ct__Q23EGG12TBitFlag<Uc>FUc (func,weak) found in eggND.a D:\home\Project\RevoYawaraka\egg\egg\modules\core\src\eggColorFader.o
```

Reply:
This is known, and is missing from other classes until `TBitFlag` infrastructure is implemented.